src/cpprealm/app.cpp

```diff
@@ -413,7 +422,7 @@ namespace realm {
             client_config.base_file_path = std::filesystem::current_path().make_preferred().generic_string();
         }
 #endif
-        client_config.user_agent_binding_info = "RealmCpp/0.0.1";
+        client_config.user_agent_binding_info = std::string("RealmCpp/") + std::string(REALM_VERSION_STRING);
```

Comment:
I don't think we should be using `REALM_VERSION_STRING` here as that's used by Core's version reporting. Neither should we clobber it in our `CMakeLists.txt` because doing that obscures the Core version string in Core's log and assertion failure messages.

Reply:
I'll create a new `REALM_CPP_SDK_VERSION`